tensorflow/core/platform/posix/posix_file_system.cc

```diff
@@ -276,4 +277,46 @@ Status PosixFileSystem::RenameFile(const string& src, const string& target) {
   return result;
 }
 
+Status PosixFileSystem::CopyFile(const string& src, const string& target) {
+#if defined(PLATFORM_WINDOWS)
```

Comment:
Why do you need to handle this case? From what I understand this case should be handled by platform/windows/windows_file_system

Reply:
Removed. Thanks.